packages/stream_video/lib/src/core/client_state.dart

```diff
@@ -26,24 +29,28 @@ abstract class ClientState {
   StateEmitter<Call?> get outgoingCall;
 
   Future<void> setOutgoingCall(Call? call);
-  Future<void> setActiveCall(Call? call);
+
+  Future<void> setActiveCall(Call call);
```

Comment:
If we want to prevent breaking changes we could allow setting `null` here still, and just cancel all active calls in that case.